2996_reflection/reflection.md

```diff
@@ -3912,6 +4178,14 @@ consteval info type_of(info r);
 
 [#]{.pnum} *Returns*: A reflection of the type of that entity.  If every declaration of that entity was declared with the same type alias (but not a template parameter substituted by a type alias), the reflection returned is for that alias.  Otherwise, if some declaration of that entity was declared with an alias it is unspecified whether the reflection returned is for that alias or for the type underlying that alias. Otherwise, the reflection returned shall not be a type alias reflection.
 
+```cpp
+consteval info value_of(info r);
+```
+
+[#]{.pnum} *Mandates*: `r` is a reflection designating either an object usable in constant expressions ([expr.const]), an enumerator, or a value.
+
+[#]{.pnum} *Returns*: If `r` is a reflection designating an object, then a reflection of the value held by that object. Otherwise, if `r` is a reflection of an enumerator, then a reflection of the value of the enumerator. Otherwise, `r`. The reflected value shall have the type of the entity reflected by `r` if it is of class type, and otherwise the cv-unqualified version of that type. The type of the reflected value shall not be an alias.
```

Comment:
As noted above, maybe something like this:

```suggestion
[#]{.pnum} *Returns*: If `r` is a reflection designating an object `o`, then a reflection of the value held by that object. The reflected value has type `delias(type_of(o)`, with the cv-qualifiers removed if this is a scalar type. Otherwise, if `r` is a reflection of an enumerator, then a reflection of the value of the enumerator. Otherwise, `r`. ```

Reply:
Done.